scss/detail.scss

```diff
@@ -0,0 +1,29 @@
+.base-span {
+  margin-left: 10px;
+  color: $text-grey;
```

Comment:
suggestion: colors name shall take the name own stand for itself. There exist a page for that. 
http://chir.ag/projects/name-that-color/#6195ED

Reply:
Namely, the property **color** is declarative for its use.